hackathon_site/dashboard/frontend/src/components/teamDetail/TeamCheckoutOrderTable/TeamCheckoutOrderTable.tsx

```diff
@@ -0,0 +1,212 @@
+                                                        hardware[row.id]?.picture ??
+                                                        hardwareImagePlaceholder
+                                                    }
+                                                    alt={hardware[row.id]?.name}
+                                                />
+                                            </TableCell>
+                                            <TableCell>
+                                                {hardware[row.id]?.name}
+                                            </TableCell>
+                                            <TableCell align={"center"}>
+                                                <IconButton size={"small"}>
+                                                    <Info
+                                                        style={{
+                                                            marginLeft: "-30px",
+                                                        }}
+                                                    />
+                                                </IconButton>
+                                            </TableCell>
+                                            <TableCell>
+                                                {`${
+                                                    hardware[row.id]?.quantity_available
+                                                }`}
+                                            </TableCell>
+                                            <TableCell>
+                                                <div
+                                                    style={{
+                                                        display: "flex",
+                                                        alignItems: "end",
+                                                    }}
+                                                >
+                                                    <Link
+                                                        underline="always"
+                                                        color="textPrimary"
+                                                        style={{ marginRight: "15px" }}
+                                                    >
+                                                        All
+                                                    </Link>
+                                                    <FormControl
+                                                        variant="standard"
+                                                        hiddenLabel={true}
+                                                    >
+                                                        <Select
+                                                            label="Qty"
+                                                            labelId="qtyLabel"
+                                                            name="quantity"
```

Comment:
Reply to your question you asked me on Slack regarding setting the default value. I have tested the correct method and it works. Here is the link for your reference: https://v4.mui.com/api/select/

I would like you to learn more and explore, and I don't want to tell you the solution directly bcz you won't learn anything by that. lmk if you still got questions

Reply:
I have set the default value to 1 on my local device, and I believe you can do that, too
<img width="1161" alt="CleanShot 2022-07-20 at 00 03 36@2x" src="https://user-images.githubusercontent.com/36619969/179894297-b297a89a-2c8b-4d00-8c07-4f6c3e3ed713.png">
!

Reply:
This issue was resolved when I changed a formik form. It doesn't look exactly like the figma anymore, but I think that is just how the formik UI looks, if you know anyway to change it, I'm more than happy to try it out.

Reply:
Can you elaborate more on this after you have resolved my last conversation? I am assuming that you are referring to the same problem or not.